### PR TITLE
Ed/blog design stance

### PR DESCRIPTION
**Pull request by urcades**
Final editing pass!

### CODE REVIEW

**Comment by urcades** [comment]
@matildepark I actually don't know if these needs a final review from you since you combed it over, just keeping good with our Process.

**Comment by urcades** [comment]
Noting I left the description out and proper date in metadata, fixing...

File: content/blog/infrastructural.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## Products
+
+Urbit’s interfaces, and the process by which we build them, are resolutely not projections of the world’s current conception of digital design—that of mass digital product design. Products are sold, bought, and discarded as they are in their physical sense. Digital products are slotted into the [vending machines](https://urcad.es/writing/personal-computer-vending-machine/) we call our phones and are mass produced as candy is produced. Candy is *essentially* free, and contemporary digital products are *absolutely* free of cost. If you’re reading this, you probably already have a good sense of what free corporate digital products mean for your privacy.
+
+Urbit is *distinctly* not free (as in freely offered as a service via a corporate entity), and your urbit is *distinctly* not a product. As we trace over the interfaces we design with our cursors, we’re not building a product, we’re not selling a product, we’re not painting a product, and we’re certainly not decorating a product.
```
Comment (matildepark) [inline content/blog/infrastructural.md]: Stress should be on “not,” not “distinctly,” imo

Comment (urcades) [reply content/blog/infrastructural.md]: Ah, yes I like this better

File: content/blog/infrastructural.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+All typography and typographic representation was severely flattened. This decision was one of our more drastic ones, and granted OS 1 a sense of being somewhat more "undirected" in the sense of its form. We hope that as our system becomes more robust, users can decide which elements of their systems require emphasizing, to feel most at home with their information.
+The majority of graphics used throughout the system were rendered into black and white line drawings. As architectural schematics and orthographic projections flatten their representation of reality so an architect can “see structure” easier, we made the decision to express a visual foundation that felt more like a diagram than an ergonomic user interface.
+A coupled notion of the above point is that we wanted to give our users the sense that modifying the underlying system was a simple task. Rather than present Urbit's first users with beautiful artistry and a sense that our infrastructure was to remain untouched and sacred, we decided to present a visual face to the project that felt nearly rudimentary, nothing outside the realm of editorial capability for the average person.
```
Comment (matildepark) [inline content/blog/infrastructural.md]: These should probably be list items? I recall they were before.

Comment (urcades) [reply content/blog/infrastructural.md]: True — also realising the list of "beautiful infrastructure" points need to be bolded

**Review by matildepark — APPROVED** [review]

**Pull request merged**